sample_files/problem_setting/test/testsuite/esoteric_validator/cases/noeol/input

```diff
@@ -0,0 +1,2 @@
+A a
+b
```

Comment:
Should be careful about this file: I foresee someone accidentally running some auto-formatter tool and just adding a newline to the end of the file.